src/library/scala/Predef.scala

```diff
@@ -376,15 +376,15 @@ object Predef extends LowPriorityImplicits {
     def length: Int                                     = sequenceOfChars.length
     def charAt(index: Int): Char                        = sequenceOfChars(index)
     def subSequence(start: Int, end: Int): CharSequence = new SeqCharSequence(sequenceOfChars.slice(start, end))
-    override def toString                               = sequenceOfChars mkString ""
+    override def toString                               = sequenceOfChars mkString
```

Comment:
Probably avoid postfix notation.

Reply:
Agreed.  Please avoid postfix notation.  The dot adds zero characters anyway; it's not like it helps anything to use postfix here.

Reply:
I'm not the thought leader here, but also see https://github.com/scala/scala/pull/6559 which makes it a felony.

Reply:
there's PostFix notation through the files I change in this patch. I'd prefer to leave those syntax changes for a different PR.

Reply:
That's reasonable, but you don't need to introduce _more_ postfix notation where there was only infix notation before.  Boyscout principle and all.

Reply:
that's 1 less postfix notation already!

Reply:
postfix vs infix. At least something was a fix.